content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -8,7 +8,11 @@ reviewers:
 - smarterclayton
 title: StatefulSets
 content_type: concept
+description: >-
+  A StatefulSet runs a group of Pods, and maintain a sticky identity for each of those Pods. This is useful for managing
```

Comment:
@sftim . The changes look good, though, can you check this phrase:
`and maintain a sticky identity for each of those Pods.`

Reply:
What do I need to check here?

Reply:
A StatefulSet runs a group of Pods, and maintains a sticky identity for each of those Pods.